lib/active_model_serializers.rb

```diff
@@ -1,32 +1,22 @@
 require 'active_model'
 require 'active_support'
-require 'action_controller'
-require 'action_controller/railtie'
+require 'active_support/core_ext/object/with_options'
 module ActiveModelSerializers
-  mattr_accessor(:logger) { ActiveSupport::TaggedLogging.new(ActiveSupport::Logger.new(STDOUT)) }
-
-  def self.config
-    ActiveModel::Serializer.config
-  end
-
   extend ActiveSupport::Autoload
   autoload :Model
   autoload :Callbacks
   autoload :Deserialization
   autoload :Logging
-end
 
-require 'active_model/serializer'
-require 'active_model/serializable_resource'
-require 'active_model/serializer/version'
+  class << self; attr_accessor :logger; end
+  self.logger = ActiveSupport::TaggedLogging.new(ActiveSupport::Logger.new(STDOUT))
 
-require 'action_controller/serialization'
-ActiveSupport.on_load(:action_controller) do
-  ActiveSupport.run_load_hooks(:active_model_serializers, ActiveModelSerializers)
-  include ::ActionController::Serialization
-  ActionDispatch::Reloader.to_prepare do
-    ActiveModel::Serializer.serializers_cache.clear
+  def self.config
+    ActiveModel::Serializer.config
   end
-end
 
-require 'active_model/serializer/railtie'
+  require 'active_model/serializer/version'
```

Comment:
https://github.com/rails-api/active_model_serializers/commit/fc04b1400a623b3fefc95bc1f5e5672ccbf61a8a but I don't remember exactly what the failures were